sources/launcher/Stride.Launcher/ViewModels/RecentProjectViewModel.cs

```diff
@@ -38,23 +38,15 @@ internal RecentProjectViewModel(LauncherViewModel launcher, UFile path)
         }
 
         public string Name { get; private set; }
-
```

Comment:
I believe it's a good practice to leave a blank line between members in a class.